sklearn/linear_model/ridge.py

```diff
@@ -847,8 +847,10 @@ def fit(self, X, y, sample_weight=None):
         -------
         self : Returns self.
         """
+        alphas = np.asarray(self.alphas)
```

Comment:
I am afraid that you have to help me with test. I am not sure of what to test and how to do it.